src/data/staff.json

```diff
@@ -90,6 +85,33 @@
       "name": "Egoizumu",
       "pfp": "https://cdn.discordapp.com/guilds/605859344243884081/users/197950963036717056/avatars/7c58b8c5a1b121a277ebe280cd684eb8.webp?size=128",
       "bio": "I am a self taught front-end developer. I mainly work with React."
+    },
+    {
+      "name": "Alex C",
+      "pfp": "https://cdn.discordapp.com/avatars/393152368318873609/7984efe0af0d2e10ea52ec5e6898f71e.png?size=1024",
+      "bio": "Code counselor"
+    }
+  ],
+  "emeritus": [
+    {
+      "name": "8-Bit",
+      "pfp": "https://cdn.discordapp.com/avatars/711997187290300546/4d491b9f6f5ff4b4465dd0bb325d2ca0.webp?size=128",
+      "bio": "Texas rancher, hobbyist programmer, occasional writer, and martial artist. I'm a retired moderator for the Codecademy Discord and have great memories working with them. Though I've stepped down, you may still see me answering questions there and in other programming communities. I seek to learn and help others were I can along the way."
+    },
+    {
+      "name": "Elisabeth (she/her)",
```

Comment:
The image link for Elisabeth's avatar does not work. But it seems to be working if '?size=1024' is removed from the string. I am not sure if it needs another size target or if removing it is a better option.

Reply:
Huh that's odd, for some strange reason (as much as I hate to say "it works on my machine!") it appears to work as it currently is just fine:
![image](https://user-images.githubusercontent.com/78568904/170117682-b119c9be-b80b-4b5b-b555-a693a1f69684.png)
And removing the size target causes it to 404 for me:
![image](https://user-images.githubusercontent.com/78568904/170117880-0628f978-5355-4498-8861-b5239bfc0856.png)


I'll play around with it a little and see if I can get something that works on both ends!